### PR TITLE
feat(designer): Parse Text action consumption support

### PR DESCRIPTION
**Pull request by bjbennet**
## Requirement Checklist

* [ ] The commit message follows our guidelines
* [ ] Tests for the changes have been added (for bug fixes or features)
* [ ] Docs have been added or updated (for bug fixes or features)

## Type of Change

* [ ] Bug fix
* [x] Feature
* [ ] Other

## Current Behavior
Consumption current does not support the action type ParseText

## New Behavior
This change adds support for the action ParseText in the designer for Consumption

## Impact of Change

* [ ] **This is a breaking change.**

## Screenshots or Videos (if applicable)
image

### CODE REVIEW

File: libs/logic-apps-shared/src/designer-client-services/lib/base/operations/dataoperations.ts [hunk 1]
```diff
@@ -42,6 +42,29 @@ export const parseJsonOperation = {
...
+  id: 'chunkText',
+  type: 'Chunktext',
+  properties: {
+    api: {
+      id: 'connectionProviders/dataOperationNew',
+      name: 'dataOperationNew',
+      brandColor: '#8c6cff',
+      description: 'Data operations',
+      displayName: 'Data operations',
+      iconUri: 'https://logicappsv2resources.blob.core.windows.net/icons/compose.svg',
```
Comment (preetriti1) [inline libs/logic-apps-shared/src/designer-client-services/lib/base/operations/dataoperations.ts:56]: Does this have compose icon only

File: libs/logic-apps-shared/src/designer-client-services/lib/base/operations/dataoperations.ts [hunk 2]
```diff
@@ -42,6 +42,29 @@ export const parseJsonOperation = {
   },
 };
 
+export const chunktext = {
+  name: 'chunktext',
+  id: 'chunkText',
+  type: 'Chunktext',
```
Comment (preetriti1) [inline libs/logic-apps-shared/src/designer-client-services/lib/base/operations/dataoperations.ts:48]: I would suggest to keep type similar to OperationType and id similar to name or follow what casing is used in other builtins

File: libs/logic-apps-shared/src/designer-client-services/lib/consumption/manifests/chunktext.ts [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+      properties: {
+        body: {
+          type: 'object',
+          title: 'Chunked result',
+          description: 'An object containing chunked text.',
+          properties: {
+            value: {
+              type: 'array',
+              title: 'Text items',
+              description: 'Array of chunked text.',
```
Comment (preetriti1) [inline libs/logic-apps-shared/src/designer-client-services/lib/consumption/manifests/chunktext.ts:94]: Does the array have any particular schema or is it just type any?

File: libs/logic-apps-shared/src/designer-client-services/lib/consumption/manifests/chunktext.ts [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+    settings: {
+      secureData: {
+        options: {
+          outputsMode: OutputSecureDataMode.LinkedToInputs,
+        },
+      },
+      trackedProperties: {
+        scopes: [SettingScope.Action],
+      },
+    },
```
Comment (preetriti1) [inline libs/logic-apps-shared/src/designer-client-services/lib/consumption/manifests/chunktext.ts:119]: Please make sure you have all settings which you require here

**Review by preetriti1 — CHANGES_REQUESTED** [review]

File: libs/logic-apps-shared/src/designer-client-services/lib/base/operations/dataoperations.ts (hunk 1 above)
Comment (bjbennet) [reply libs/logic-apps-shared/src/designer-client-services/lib/base/operations/dataoperations.ts:56]: We are having a custom icon created, but for now it shares the same icon as all the other icons under Data Operations.

File: libs/logic-apps-shared/src/designer-client-services/lib/consumption/manifests/chunktext.ts (hunk 3 above)
Comment (bjbennet) [reply libs/logic-apps-shared/src/designer-client-services/lib/consumption/manifests/chunktext.ts:94]: its just an array of strings

File: libs/logic-apps-shared/src/designer-client-services/lib/consumption/manifests/chunktext.ts (hunk 4 above)
Comment (bjbennet) [reply libs/logic-apps-shared/src/designer-client-services/lib/consumption/manifests/chunktext.ts]: adjust some settings

**Review by preetriti1 — CHANGES_REQUESTED** [review]

File: libs/logic-apps-shared/src/designer-client-services/lib/consumption/manifests/chunktext.ts (hunk 3 above)
Comment (bjbennet) [reply libs/logic-apps-shared/src/designer-client-services/lib/consumption/manifests/chunktext.ts]: updated

**Review by preetriti1 — APPROVED** [review]

**Pull request merged**